content/docs/deployment/csminstallationwizard/_index.md

```diff
@@ -69,23 +69,25 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 3. Look over all the fields in the generated `values.yaml` and fill in/adjust any as needed.
 
-4. When the driver is installed using values generated by installation wizard, then the user needs to update the secret for driver by patching the MDM keys, as follows:
+4. When the PowerFlex driver is installed using values generated by installation wizard, then the user needs to update the secret for driver by patching the MDM keys, as follows:
 
 **Steps**
 * `echo -n '<MDM_IPS>' | base64`
 * `kubectl patch secret vxflexos-config -n vxflexos -p "{\"data\": { \"MDM\": \"<GENERATED_BASE64>\"}}"`
 
-5. For the Observability module, please refer [Observability](../../observability/deployment/#post-installation-dependencies) to install the post installation dependencies.
+5. If Observability is enabled, please refer to [Observability](../../observability/deployment/#post-installation-dependencies) to install the post installation dependencies.
```

Comment:
change to " If Observability is enabled, please refer to [Observability](../../observability/deployment/#post-installation-dependencies) for post installation dependencies.